bokeh/models/widgets/icons.py

```diff
@@ -35,3 +35,8 @@ class Icon(AbstractIcon):
     Indicates a spinning (animated) icon. This value is ignored for
     icons that do not support spinning.
     """)
+
+    spin_updates = Int(0, help="""
+    This is a dummy field for generated a second callback if the spin state has
```

Comment:
this comment could probably made more clear.. If an user is just reading this without a context I'm not sure they'll understand what "callback" it's talking about